src/main/java/uk/gov/digital/ho/hocs/audit/utils/UuidStringChecker.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.digital.ho.hocs.audit.utils;
+
+import org.springframework.util.StringUtils;
+
+import java.util.regex.Pattern;
+
+public class UuidStringChecker {
+
+    private static final String UUID_REGEX = "^[0-9a-f]{8}\\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\\b[0-9a-f]{12}$";
```

Comment:
As we already have this defined in workflow, wonder whether we look at pulling this out into a package and consuming in both modules

Reply:
We use this in plenty of locations - will raise a ticket to discuss this as I think a package for UUID testing is likely a little overkill.